dashboard/test/models/script_test.rb

```diff
@@ -2387,6 +2387,34 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal expected_levels, actual_levels
   end
 
+  test 'accessing lessons through lesson groups is same as directly from script' do
```

Comment:
Can you add two levels in one of the lessons?

Reply:
done